content/integrations/java.md

```diff
@@ -103,7 +103,7 @@ You can explicitly specify supplementary tags. For instance, assuming the follow
 
     mydomain:attr0=val0,attr1=val1
 
-It would create a metric called `mydomain` (or some variation depending on the attribute inside the bean) with tags: `attr0:val0, attr1:val1, domain:mydomain, simple:val0, raw_value:my_chosen_value, multiple:val0-val1`. 
```

Comment:
Thanks for the several corrections you added to the text. 

For this commit though, I believe those 3 tags should not be removed: they are part of the tags that would be added by the jmxfetcher in that case. Actually, they are the whole point of my pull request :) 

What happens is that since https://github.com/DataDog/jmxfetch/pull/117 got merged, we can add tags at the level  of each Mbean, as shown by the example lines I added to the example configuration: 

```
                  tags:
                         simple: $attr0
                         raw_value: my_chosen_value
                         multiple: $attr0-$attr1
```

And those tags get added to the tags of the Datadog metric: in our case they get translated into `simple:val0, raw_value:my_chosen_value, multiple:val0-val1`. I like this example since it also shows how `$attr0-$attr1` get replaced with values from the MBean. 

I tested that behaviour on my laptop with the release 0.15 of jmxfetcher and that's indeed what I observe.